spec.html

```diff
@@ -14980,7 +14981,8 @@ <h1>Module Namespace Exotic Objects</h1>
               a List of Strings
             </td>
             <td>
-              A List whose elements are the String values of the exported names exposed as own properties of this object. The list is ordered as if an Array of those String values had been sorted using %Array.prototype.sort% using *undefined* as _comparefn_.
+              A List whose elements are the String values of the exported names exposed as own properties of this object.
+              The list is sorted according to lexicographic code unit order.
```

Comment:
We don't typically have a single paragraph span multiple lines like this. We just put everything on one big long line.